README.md

```diff
@@ -164,6 +164,50 @@ In order to use "Unbreakable Selenium", you just need to wrap your usual driver
  re = new RecheckWebImpl();
 ```
 
+## Usage of rehub
```

Comment:
I know it's not your change, but row 157 'The code would _the_ look like so' seems to be wrong.